test/module/transport-builtin-test.js

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
```

Comment:
Any reason to use .js instead of .ts for this file?

Reply:
SharedArrayBuffer can't is not supported by current ts compiler. 

---
In reply to: [155906850](https://github.com/Microsoft/napajs/pull/137#discussion_r155906850) [](ancestors = 155906850)

Reply:
test/module is used for Napa module testing.

I guess you can have this file merged into transport-test.ts?

Reply:
How can we test lifecycle?

Reply:
#150 to track it change it ts.

---
In reply to: [155906994](https://github.com/Microsoft/napajs/pull/137#discussion_r155906994) [](ancestors = 155906994,155906850)